src/sentry/conf/server.py

```diff
@@ -1236,13 +1236,15 @@ def create_partitioned_queues(name):
 SENTRY_USE_X_FORWARDED_FOR = True
 
 SENTRY_DEFAULT_INTEGRATIONS = (
+    'sentry.integrations.bitbucket.BitBucketIntegrationProvider',
```

Comment:
maybe just comment out this line until it's fully functional? i don't think we need to actually have this in the list if it's just going to blow up at some point in the pipeline if an internal user tries to add it.